Source/JavaScriptCore/jit/AssemblyHelpers.cpp

```diff
@@ -607,44 +607,113 @@ void AssemblyHelpers::emitAllocateWithNonNullAllocator(GPRReg resultGPR, const J
     // - We *can not* use RegisterSetBuilder::macroScratchRegisters on x86.
     // - We *can* use RegisterSetBuilder::macroScratchRegisters on ARM.
 
-    Jump popPath;
+    Jump popPath, zeroPath;
```

Comment:
Let's have `Jump zeroPath`. We normally do not use comma separated declarations.

Reply:
Resolved in latest push.